capstone/src/main/java/com/google/sps/servlets/AssociationServlet.java

```diff
@@ -58,9 +63,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
    * @param query the entities to get the names from
    * @return the arraylist to add the names of the entities to
    */
-  private ArrayList<String> extractContent(QueryResultList<Entity> query) {
+  private ArrayList<String> extractContent(QueryResultList<Entity> query, String scope) {
     ArrayList<String> output = new ArrayList<String>();
     for (Entity entity : query) {
+      if (!scope.equals((String) entity.getProperty("scope"))) continue;
+      if (output.size() >= LIMIT) break;
```

Comment:
I would rename LIMIT to something a bit more self descriptive. Also please use braces:

```
if (output.size() >= LIMIT) {
    break;
}
```

https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used

Reply:
+1 :-)